core/build.gradle

```diff
@@ -71,6 +71,7 @@ sourceSets {
             include "io/crate/core/StringUtils*"
             include "io/crate/geo/GeoJSONUtils*"
             include "io/crate/types/**"
+            include "io/crate/exceptions/*"
```

Comment:
We need to add a separate exception for validating IP addresses here, so that we can distinguish between cast errors and IP validation errors. Having the IP address validation as ConversionException would just raise a `cannot cast` error otherwise. This might be confusing.

The new exception type has to go into the core, since the types are there and the IP type validates and throws an exception there. Or am I missing something?